neptune/internal/utils/alpha_integration.py

```diff
@@ -140,12 +140,11 @@ def channelType(self):
             return ChannelType.IMAGE.value
 
     @property
-    def lastX(self):
-        # TODO: NPT-9216
-        return None
+    def x(self):
+        return self._properties.lastStep
```

Comment:
Here's no such value: `_properties.lastStep`.
None of `*SeriesAttributeDTO` contains it when returned from API.

Reply:
Fetch and look again. ;)

Reply:
Hmmm, still no `x`  value on `stage` (screen).
![image](https://user-images.githubusercontent.com/3773836/109038694-4b14ea80-76cc-11eb-8bf8-ef99ca0908a8.png)

But I assume it'll work after deploying `[e6f65a16] Add last step to series attributes ` from backend repo.